### ISSUE TITLE
faile to build with -Werror=return-type

### ISSUE THREAD

**Issue by andreasstieger** [ColumPaget/libUseful#5]
`stream.c` `static void *SelectAddFD` is declared as returning a `void *` but never does.

```
[   15s] Stream.c: In function 'SelectAddFD':
[   15s] Stream.c:56:1: error: no return statement in function returning non-void [-Werror=return-type]
[   15s]    56 | }
[   15s]       | ^
```

````
[   13s] Process.c: In function 'ProcessApplyLateConfig':
[   13s] Process.c:892:1: error: control reaches end of non-void function [-Werror=return-type]
[   13s]   892 | }
[   13s]       | ^
[   13s] cc1: some warnings being treated as errors
[   13s] make: *** [Makefile:190: Process.o] Error 1
`````

**Comment by ColumPaget** [comment]
Thanks for catching these, the second one is more than just a build error! They'll be fixed in version 5.7

**Comment by andreasstieger** [comment]
Also

````
[   37s] StreamAuth.c: In function 'STREAMAuthValueFile':
[   37s] StreamAuth.c:13:1: error: no return statement in function returning non-void [-Werror=return-type]
[   37s]    13 | }
[   37s]       | ^
[   37s] StreamAuth.c: At top level:
[   37s] StreamAuth.c:9:12: warning: 'STREAMAuthValueFile' defined but not used [-Wunused-function]
[   37s]     9 | static int STREAMAuthValueFile(const char *Path, const char *Value)
[   37s]       |            ^~~~~~~~~~~~~~~~~~~
[   37s] cc1: some warnings being treated as errors
````
